sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_operations/_operations.py

```diff
@@ -3740,1084 +2832,1323 @@ def _classify_document_initial(  # pylint: disable=inconsistent-return-statement
         response_headers["Retry-After"] = self._deserialize("int", response.headers.get("Retry-After"))
         response_headers["Operation-Location"] = self._deserialize("str", response.headers.get("Operation-Location"))
 
+        deserialized = response.iter_bytes()
+
         if cls:
-            return cls(pipeline_response, None, response_headers)  # type: ignore
+            return cls(pipeline_response, deserialized, response_headers)  # type: ignore
+
+        return deserialized  # type: ignore
 
     @overload
-    def begin_classify_document(
+    def begin_analyze_batch_documents(
         self,
-        classifier_id: str,
-        classify_request: _models.ClassifyDocumentRequest,
+        model_id: str,
+        analyze_batch_request: Optional[_models.AnalyzeBatchDocumentsRequest] = None,
         *,
+        pages: Optional[str] = None,
+        locale: Optional[str] = None,
         string_index_type: Optional[Union[str, _models.StringIndexType]] = None,
-        split: Optional[Union[str, _models.SplitMode]] = None,
+        features: Optional[List[Union[str, _models.DocumentAnalysisFeature]]] = None,
+        query_fields: Optional[List[str]] = None,
+        output_content_format: Optional[Union[str, _models.ContentFormat]] = None,
+        output: Optional[List[Union[str, _models.AnalyzeOutputOption]]] = None,
         content_type: str = "application/json",
         **kwargs: Any,
-    ) -> LROPoller[_models.AnalyzeResult]:
-        # pylint: disable=line-too-long
-        """Classifies document with document classifier.
+    ) -> LROPoller[_models.AnalyzeBatchResult]:
+        """Analyzes batch documents with document model.
 
-        :param classifier_id: Unique document classifier name. Required.
-        :type classifier_id: str
-        :param classify_request: Classify request parameters. Required.
-        :type classify_request: ~azure.ai.documentintelligence.models.ClassifyDocumentRequest
+        :param model_id: Unique document model name. Required.
+        :type model_id: str
+        :param analyze_batch_request: Analyze batch request parameters. Default value is None.
+        :type analyze_batch_request: ~azure.ai.documentintelligence.models.AnalyzeBatchDocumentsRequest
+        :keyword pages: List of 1-based page numbers to analyze.  Ex. "1-3,5,7-9". Default value is
```

Comment:
```suggestion
        :keyword pages: Range of 1-based page numbers to analyze.  Ex. "1-3,5,7-9". Default value is
```
just a nit as pages doesnt take a `list`.

Reply:
Updated across this package. Will request the change in tsp.